utils/src/requests/UpdateSubjectDTO.ts

```diff
@@ -10,7 +10,9 @@ import {
 } from '../ValidationUtil';
 
 export class UpdateSubjectDTO {
-  @ApiProperty()
+  @ApiProperty({
+    description: 'Subject name',
+  })
```

Comment:
Add IsString decorator with validation message and also add that message to documentation